assets/sass/06-components/pagination.scss

```diff
@@ -25,6 +25,11 @@
 
 	.nav-links {
 
+		> * {
```

Comment:
Just a quick question: why the direct children + universal selector pattern `> *` is used so widely across the theme's CSS? Not sure in this case there's the need to use it. Seems to me all the targeted elements do have a specific CSS class.

Reply:
Update: quickly discussed this on Slack, agreed to consider later refinements. Not a blocker.